artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPMessage.java

```diff
@@ -128,7 +130,7 @@ private void initalizeObjects() {
    }
 
    private Map getApplicationPropertiesMap() {
-
+      new Exception("trace").printStackTrace();
```

Comment:
oops.. not intended :) 

It's my usual debug...


I will have to fix this later tonight..